ReactApp/src/components/Editor.jsx

```diff
@@ -48,25 +48,19 @@ import { unlock } from './lock-unlock';
 // Current editor (assumes can be only one instance).
 let editor = {};
 
-const { useBlockEditorSettings } = unlock(editorPrivateApis);
-const {
-	ExperimentalBlockEditorProvider: BlockEditorProvider,
-	ExperimentalBlockCanvas: BlockCanvas,
-} = unlock(blockEditorPrivateApis);
+const { ExperimentalBlockCanvas: BlockCanvas } = unlock(blockEditorPrivateApis);
 
 function Editor({ post }) {
 	const [blocks, setBlocks] = useState([]);
-	const [registeredBlocks] = useState([]);
 	const [_isCodeEditorEnabled, setCodeEditorEnabled] = useState(false);
 	const titleRef = useRef();
 	const { addEntities, receiveEntityRecords } = useDispatch(coreStore);
-	const { setupEditor } = useDispatch(editorStore);
 
 	useEffect(() => {
 		window.editor = editor;
+		addEntities(postTypeEntities);
 		receiveEntityRecords('postType', post.type, post);
 
-		setupEditor(post, {});
```

Comment:
Sure! I've removed it because it is called within the [`EditorProvider`](https://github.com/WordPress/gutenberg/blob/256fa3f93e7ef70625232ade855a0dbb36e03599/packages/editor/src/components/provider/index.js#L267) component.